src/gui/windows/chat.zig

```diff
@@ -243,7 +243,7 @@ pub fn render() void {
 }
 
 pub fn addMessage(msg: []const u8) void {
-	messageQueue.enqueue(main.globalAllocator.dupe(u8, msg));
+	messageQueue.push(main.globalAllocator.dupe(u8, msg));
```

Comment:
`front` and `back` suffixes make sense only if you can use them to tell the direction the queue is moving. Therefore I am convinced that this should not be just `push` but rather (which one? Anyway, the one with suffix) so it is immediately obvious that history operates in two directions. If I didn't know how this code works, I would have to look at the implementation to determine if pushBack and this push are the same or opposite. Conventions are nice, but they don't replace descriptive names.